azure-pipelines/end-to-end-tests-dir/z-applocalcpp.ps1

```diff
@@ -0,0 +1,34 @@
+if ($IsWindows) {
+    . $PSScriptRoot/../end-to-end-tests-prelude.ps1
+

```

Comment:
With a separate 'build.bat' file, you should be able to do something like
```pwsh
    Run-Vcpkg env "$buildDir/build.bat" "$buildDir"
```
build.bat:
```cmd
cd %1
cl mylib.cpp /dll
cl myexe.cpp mylib.lib
```